website/data/language-nav-data.json

```diff
@@ -220,6 +220,21 @@
       }
     ]
   },
+  {
+    "title": "Moved block",
+    "path": "moved"
+  },
+  { "title": "Checks", "path": "checks" },
```

Comment:
Oooh, TF 1.3 didn't have checks or the import block 😓 Maybe we shouldn't backport this far out? Maybe stick with 1.5 and up?